src/main/java/com/exactpro/th2/codec/grpc/ProtoDecoder.java

```diff
@@ -0,0 +1,106 @@
+/*******************************************************************************
+ * Copyright 2020-2021 Exactpro (Exactpro Systems Limited)
```

Comment:
```suggestion
 * Copyright 2021 Exactpro (Exactpro Systems Limited)
```
The first year should be the same when the file has been created.